templates/default/rt/rtverifiers.html

```diff
@@ -4,7 +4,7 @@
 	{if !ConfigHelper::checkConfig('phpui.helpdesk_hide_disabled_users')
 		|| (ConfigHelper::checkConfig('phpui.helpdesk_hide_disabled_users') && $user.access)
 		|| $user.id == $ticket.verifierid}
-	<OPTION value="{$user.id}"{if $user.id == $ticket.verifierid} selected{/if}{if !$user.access} class="blend"{/if}>{$user.rname}</OPTION>
+	<OPTION value="{$user.id}"{if $user.id == $ticket.verifierid} selected{/if}{if !$user.access} class="blend"{/if}>{$user.name}</OPTION>
```

Comment:
Skąd nagle te zmiany?
Nie lubię jak ktoś pod szyldem jednej zmiany robi jednocześnie 5 innych zmian, bo potem ciężko się połapać, która zmiana za co odpowiadała?

Reply:
Wywaliłem z modules/rtticketadd.php bezpośrednie odwołanie do DB i zastąpiłem to funkcją która od dawna jest w systemie.

Reply:
było $user.rname, a jest $user.name. Po co taka zmiana?
Za każdym razem jest cała masa dyskusji na ten sam temat - zmian robionych mimochodem na boku :(

Reply:
Możesz wskazać na demo, gdzie zobaczę to puste miejsce?